samples/server/petstore/python-fastapi/requirements.txt

```diff
@@ -2,7 +2,7 @@ aiofiles==0.5.0
 aniso8601==7.0.0
 async-exit-stack==1.0.1
 async-generator==1.10
-certifi==2022.12.7
+certifi==2022.12.07
```

Comment:
should be 2022.12.7 according to https://pypi.org/project/certifi/#history, right?